pajbot/modules/basic/permaban.py

```diff
@@ -78,6 +94,12 @@ def unpermaban_command(self, bot, source, message, **rest):
             if self.settings["unban_from_chat"] is True:
                 bot.unban(user)
 
+            if (
+                self.settings["enable_send_timeout"] is True
+                and self.settings["unban_from_chat"] is True
+            ):
+                bot.timeout(user, 1, self.settings["timeout_reason"].format(source=source), once=True)
+
```

Comment:
```suggestion
            if self.settings["unban_from_chat"] is True:
                bot.unban(user)
                
                if self.settings["enable_send_timeout"] is True:
                    # This is delayed because you can't timeout a user immediately after
                    # unban (some caching effect on Twitch's end, the timeout errors as
                    # if the user was still banned if you don't add a delay)
                    # Delayed by 3 seconds here
                    bot.execute_delayed(
                        3,
                        bot.timeout,
                        user,
                        1,
                        self.settings["timeout_reason"].format(source=source),
                        once=True
                    )
```

Reply:
Note I didn't check the format, it might need to be reformatted before it can be merged